CHANGES/1485.removal

```diff
@@ -0,0 +1,2 @@
+Removed the optional "kid" parameter stored inside the signatures' payload generated during
```

Comment:
it's going to look a bit weird because other incoming schema1 manifests (not converted by pulp)  would contain it

Reply:
And that is perfectly fine because users can verify the origin of the fingerprint. When it comes to the on-the-fly schema conversion, there is no way the users benefit from it, is it?

Reply:
I meant that there might be incoming schema1 directly from remote registry without us doing any conversion. And that schema will contain kid. It was not a concern or anything, just an observation. I don't we should worry about this much because we've just agreed to deprecate the whole conversion mechanism